lib/puppet/provider/selinux_fcontext/semanage.rb

```diff
@@ -30,7 +30,7 @@ def self.prefetch(resources)
     execpipe("#{command(:semanage)} fcontext -lC") do |out|
       lines = out.readlines[2..-1] || []
       lines.each do |line|
-        filespec, filetype, context = line.split(/\s\s+/)
+        filespec, filetype, context = line.scan(/^(\S+)\s+(\S+\s?\S+)\s+(\S+)\s+$/).flatten
```

Comment:
\s+ matches *all* whitespaces characters. Have a look here: https://ruby-doc.org/core-2.1.1/Regexp.html

> /\s/ - A whitespace character: /[ \t\r\n\f]/

It works in production 3 years now.